include/openssl/bio.h

```diff
@@ -552,11 +552,11 @@ int BIO_get_shutdown(BIO *a);
 void BIO_vfree(BIO *a);
 int BIO_up_ref(BIO *a);
 int BIO_read(BIO *b, void *data, int dlen);
-int BIO_read_ex(BIO *b, void *data, size_t dlen, size_t *read);
-int BIO_gets(BIO *bp, char *buf, int size);
+int BIO_read_ex(BIO *b, void *data, size_t dlen, size_t *readbytes);
+int BIO_gets(BIO *bp, char *out, int outl);
```

Comment:
No, please avoid using `out` and `in` in the BIO input and output routines.  It's confusing, as to some people (such as myself), `out` signifies something that goes *out* (i.e. not something you read) and `in` signifies something that comes *in* (i.e. not something you write).

So please revert the `BIO_gets()` change.